plugins/FreeBoy/FreeBoy.cpp

```diff
@@ -236,188 +236,192 @@ f_cnt_t FreeBoyInstrument::desiredReleaseFrames() const
 		}
-		papu->writeRegister( 0xff18, data & 0xff );
-		papu->writeRegister( 0xff19, (data>>8) | initflag );
-		papu->writeRegister( 0xff1d, data & 0xff );
-		papu->writeRegister( 0xff1e, (data>>8) | initflag );
+		papu->writeRegister(0xff18, data & 0xff);
+		papu->writeRegister(0xff19, (data >> 8) | initflag);
+		papu->writeRegister(0xff1d, data & 0xff);
+		papu->writeRegister(0xff1e, (data >> 8) | initflag);
 	}
 
-	if( tfp == 0 )
+	if (tfp == 0)
 	{
 		//PRNG Frequency = (1048576 Hz / (ratio + 1)) / 2 ^ (shiftclockfreq + 1)
-		char sopt=0;
-		char ropt=1;
-		float fopt = 524288.0 / ( ropt * pow( 2.0, sopt + 1.0 ) );
+		char sopt = 0;
+		char ropt = 1;
+		float fopt = 524288.0 / (ropt * pow(2.0, sopt + 1.0));
 		float f;
-		for ( char s=0; s<16; s++ )
-		for ( char r=0; r<8; r++ ) {
-			f = 524288.0 / ( r * pow( 2.0, s + 1.0 ) );
-			if( fabs( freq-fopt ) > fabs( freq-f ) ) {
-				fopt = f;
-				ropt = r;
-				sopt = s;
+		for (char s = 0; s < 16; ++s)
+		{
+			for (char r = 0; r < 8; ++r)
+			{
+				f = 524288.0 / (r * pow(2.0, s + 1.0));
+				if (fabs(freq - fopt) > fabs(freq - f))
+				{
+					fopt = f;
+					ropt = r;
+					sopt = s;
+				}
 			}
 		}
+
 		data = sopt;
 		data = data << 1;
 		data += m_ch4ShiftRegWidthModel.value();
 		data = data << 3;
 		data += ropt;
-		papu->writeRegister( 0xff22, data );
+		papu->writeRegister(0xff22, data);
 
 		//channel 4 init
-		papu->writeRegister( 0xff23, 128 );
+		papu->writeRegister(0xff23, 128);
 	}
 
-	int const buf_size = 2048;
+	constexpr int bufSize = 2048;
 	int framesleft = frames;
 	int datalen = 0;
-	auto buf = std::array<blip_sample_t, buf_size * 2>{};
-	while( framesleft > 0 )
+	auto buf = std::array<blip_sample_t, bufSize * 2>{};
+	while (framesleft > 0)
 	{
 		int avail = papu->samplesAvail();
-		if( avail <= 0 )
+		if (avail <= 0)
 		{
 			papu->endFrame(FRAME_LENGTH);
 			avail = papu->samplesAvail();
 		}
-		datalen = framesleft>avail?avail:framesleft;
-		datalen = datalen>buf_size?buf_size:datalen;
+		datalen = framesleft > avail ? avail : framesleft;
+		datalen = datalen > bufSize ? bufSize : datalen;
 
 		long count = papu->readSamples(buf.data(), datalen * 2) / 2;
 
-		for( fpp_t frame = 0; frame < count; ++frame )
+		for (fpp_t frame = 0; frame < count; ++frame)
 		{
-			for( ch_cnt_t ch = 0; ch < DEFAULT_CHANNELS; ++ch )
+			for (ch_cnt_t ch = 0; ch < DEFAULT_CHANNELS; ++ch)
 			{
-				sample_t s = float(buf[frame*2+ch])/32768.0;
-				_working_buffer[frames-framesleft+frame+offset][ch] = s;
+				sample_t s = static_cast<float>(buf[(frame * 2) + ch]) / 32768.0f;
+				workingBuffer[frames - framesleft + frame + offset][ch] = s;
 			}
 		}
 		framesleft -= count;
 	}
-	instrumentTrack()->processAudioBuffer( _working_buffer, frames + offset, _n );
+	instrumentTrack()->processAudioBuffer(workingBuffer, frames + offset, nph);
 }
 
 
 
-void FreeBoyInstrument::deleteNotePluginData( NotePlayHandle * _n )
+void FreeBoyInstrument::deleteNotePluginData( NotePlayHandle * nph )
```

Comment:
Style

Reply:
You can click the "Files Changed" tab in GitHub to see all my changes together before writing review comments. Looking at this PR commit-by-commit is just helpful for understanding what kinds of changes I made to the code, because the changes made in the 1st commit are what fix the bug, the 2nd commit does some refactoring, the 3rd fixes most of the style, etc.

Line 253 of FreeBoy.cpp is the only place I know of where I don't adhere to the style guidelines, and that was deliberate because I have another open PR which fixes that line and I want to get it into master right after this PR.